sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/api/Azure.MixedReality.ObjectAnchors.Conversion.netstandard2.0.cs

```diff
@@ -102,6 +97,45 @@ public partial class AssetUploadUriResult
+        public static Azure.MixedReality.ObjectAnchors.Conversion.ConversionErrorCode InvalidGravity { get { throw null; } }
+        public static Azure.MixedReality.ObjectAnchors.Conversion.ConversionErrorCode InvalidJobId { get { throw null; } }
+        public static Azure.MixedReality.ObjectAnchors.Conversion.ConversionErrorCode InvalidScale { get { throw null; } }
+        public static Azure.MixedReality.ObjectAnchors.Conversion.ConversionErrorCode NoError { get { throw null; } }
+        public static Azure.MixedReality.ObjectAnchors.Conversion.ConversionErrorCode ServiceError { get { throw null; } }
+        public static Azure.MixedReality.ObjectAnchors.Conversion.ConversionErrorCode TooManyRigPoses { get { throw null; } }
+        public static Azure.MixedReality.ObjectAnchors.Conversion.ConversionErrorCode Unknown { get { throw null; } }
+        public static Azure.MixedReality.ObjectAnchors.Conversion.ConversionErrorCode ZeroFaces { get { throw null; } }
+        public static Azure.MixedReality.ObjectAnchors.Conversion.ConversionErrorCode ZeroTrajectoriesGenerated { get { throw null; } }
+        public bool Equals(Azure.MixedReality.ObjectAnchors.Conversion.ConversionErrorCode other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.MixedReality.ObjectAnchors.Conversion.ConversionErrorCode left, Azure.MixedReality.ObjectAnchors.Conversion.ConversionErrorCode right) { throw null; }
+        public static implicit operator Azure.MixedReality.ObjectAnchors.Conversion.ConversionErrorCode (string value) { throw null; }
+        public static bool operator !=(Azure.MixedReality.ObjectAnchors.Conversion.ConversionErrorCode left, Azure.MixedReality.ObjectAnchors.Conversion.ConversionErrorCode right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class NotSupportedAssetFileTypeException : System.Exception, System.Runtime.Serialization.ISerializable
+    {
+        public NotSupportedAssetFileTypeException() { }
+        protected NotSupportedAssetFileTypeException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
+        public NotSupportedAssetFileTypeException(string message) { }
+        public NotSupportedAssetFileTypeException(string message, System.Exception inner) { }
+        public Azure.MixedReality.ObjectAnchors.Conversion.AssetFileType AttemptedFileType { get { throw null; } }
+        public System.Collections.Generic.IEnumerable<Azure.MixedReality.ObjectAnchors.Conversion.AssetFileType> SupportedAssetFileTypes { get { throw null; } }
```

Comment:
It is interesting that this is an IEnumerable. Any reason not to use IReadOnlyList here?

Reply:
I'm not sure there are any meaningful scenarios for indexing into this collection?  I expect the only thing customers will do is pass these values to `string.Join`.

Reply:
Fair point, but what about IReadOnlyCollection?

Reply:
It's a less familiar interface and only adds `Count` on top of `IEnumerable`.  I'm not sure there's much value because I don't see customers actually using the `Count` property either.  I'd stick with either `IEnumerable` or `IReadOnlyList` for everything immutable.  I don't feel strongly here, but based on expected customer usage I think `IEnumerable` makes the most sense.